sql/report_queries/items_lost_in_transit/items_lost_in_transit.go

```diff
@@ -0,0 +1,13 @@
+package items_lost_in_transit
+
+import (
+	"testing"
+
+	"github.com/folio-org/folio-analytics/gotest"
+)
+
+func TestQuery(t *testing.T) {
+
+	gotest.RunTest(t, "items_lost_in_transit.sql", "items_lost_in_transit_result.csv")
+
+}
```

Comment:
I assume this is for local test purposes. Is it supposed to be part of the repository?

Reply:
so, this is part of an old automated testing scheme. it may not be necessary now, but I've continued to do it for my report queries. @nassibnassar, would you like me to remove the testing files?

Reply:
@amzoss Yes, I think they should be removed.  The testing model requires static test data which we are no longer able to maintain.

Reply:
Excellent! Latest commit has removed those files.